desktop-widgets/mainwindow.cpp

```diff
@@ -1159,24 +1166,17 @@ void MainWindow::on_actionViewAll_triggered()
 {
 	toggleCollapsible(false);
 	beginChangeState(VIEWALL);
-	static QList<int> mainSizes;
+
 	const int appH = qApp->desktop()->size().height();
 	const int appW = qApp->desktop()->size().width();
-	if (mainSizes.empty()) {
-		mainSizes.append(lrint(appH * 0.7));
-		mainSizes.append(lrint(appH * 0.3));
-	}
-	static QList<int> infoProfileSizes;
-	if (infoProfileSizes.empty()) {
-		infoProfileSizes.append(lrint(appW * 0.3));
-		infoProfileSizes.append(lrint(appW * 0.7));
-	}
 
-	static QList<int> listGlobeSizes;
-	if (listGlobeSizes.empty()) {
-		listGlobeSizes.append(lrint(appW * 0.7));
-		listGlobeSizes.append(lrint(appW * 0.3));
-	}
+    auto _lrint = [](double value) {
+        return static_cast<int>(std::lrint(value));
+    };
+
+    auto mainSizes = QList<int>{ _lrint(appH * 0.7), _lrint(appH * 0.3) };
```

Comment:
What's in this case the advantage of `auto` over the more traditional
```
QList<int> mainSizes = { rint(appH * 0.7), rint(appH * 0.3) };
```
?
Seems only more characters to type..?

Reply:
thank you - I was wondering the same but couldn't quite figure out how to ask

Reply:
no advantage whatsoever. I'm used doing that because of my work and you know how muscle memory is, I'm using the variant without auto.